app/main/posts/views/filters/filter-transformers.service.js

```diff
@@ -9,21 +9,23 @@ function FilterTransformersService(_, FormEndpoint, TagEndpoint, RoleEndpoint,
     this.rawFilters = {};
 
     this.requestsFiltersData = function () {
-        return $q.all([
-                RoleEndpoint.query().$promise,
-                UserEndpoint.query().$promise,
-                TagEndpoint.query().$promise,
-                FormEndpoint.query().$promise,
-                SavedSearchEndpoint.query({}).$promise,
-                CollectionEndpoint.query({}).$promise
-            ]).then(function (results) {
-            roles = _.indexBy(results[0], 'name');
-            users = _.indexBy(results[1], 'id');
-            tags = _.indexBy(results[2], 'id');
-            forms = _.indexBy(results[3], 'id');
-            savedSearches = _.indexBy(results[4], 'id');
-            collections = _.indexBy(results[5], 'id');
-        });
+        if (PostMetadataService.validateUser) {
```

Comment:
Ok absolutely last thing :D You need to call the validateUser function by adding () after it like this: `PostMetadataService.validateUser()`, otherwise it just checks if the function is there or not...